src/components/AdminDashboard/CompleteIncident.js

```diff
@@ -53,6 +55,22 @@ const CompleteIncident = props => {
     setEditing(!editing);
   };
 
+  // for incident deletion button
+  const toggleDelete = evt => {
```

Comment:
Does this actually toggle deleting? It looks like you never call setDeleting. Does the button just delete without confirmation?

Reply:
Currently the button does delete without an intermediate step. As the functionality is now working, a two step process is the next step.

Reply:
It seems this function could be renamed to something like "handleDelete" rather than "toggleDelete" based on functionality. 

I agree with Kelly's suggestion of a confirmation. It seems there were previous efforts toward this in components > AdminDashboard > ApprovedIncident.js, that may be worth exploring.

Reply:
So Luis & Naj are going to work on creating a confirmation modal in another PR, but since the MVP functionality is fleshed out here, I recommended pushing these commits thus far